src/components/TextPopup/TextPopup.js

```diff
@@ -114,6 +114,14 @@ const TextPopup = () => {
           }
           dataElement="textStrikeoutToolButton"
         />
+        <ActionButton
```

Comment:
Do you want to make this as a separate component, say `LinkButton`, and import this button in `AnnotationPopup` and `TextPopup`? This way we can avoid some duplication. I'm ok with what we have now.

Reply:
I think ActionButton is fine, since we don't anticipate it changing much and not much different from other action button. Otherwise we would duplicate code and create action button, redaction button, link button?

Reply:
Hmm I meant to say that we can create a LinkButton component that renders an ActionButton and use that LinkButton in the two Popup components. So if we ever change the props to this LinkButton we just need to change 1 place. But just like what you said we don't expect this button to be changed much so what we have is totally Okay